bigflow/log.py

```diff
@@ -127,7 +130,13 @@ def init_logging(config: LogConfigDict, workflow_id: str):
         'run_uuid': run_uuid,
     }
 
+    # clean all existings handlers
+    root = logging.getLogger(None)
```

Comment:
I'm not sure if we want to clean the existing configuration. For example, Dataflow provides nice logging UI utilities and we want to keep them. 

I suggest making BigFlow logging as an extension (so BigFlow logging configuration sends logs to an additional place).

Reply:
It makes sense.
I've changed logic to do 2 things:
 - install 'stderr' logger only when no other loggers are installed (otherwise most probably either console logger is already installed OR is not needed at all)
 - update root log-level if it greater than specified log_level.